modules/yugabytedb/options.go

```diff
@@ -0,0 +1,53 @@
+package yugabytedb
+
+import (
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// WithYSQLDatabaseName sets the initial database name for the yugabyteDB container.
+func WithYSQLDatabaseName(ysqlDBName string) testcontainers.CustomizeRequestOption {
```

Comment:
question: this seems overly verbose, do we need YSQL qualification, similar with the other functional options.
```suggestion
// WithDatabaseName sets the initial database name for the yugabyteDB container.
func WithDatabaseName(ysqlDBName string) testcontainers.CustomizeRequestOption {
```

Reply:
Attended ✅